docs/running.rst

```diff
@@ -0,0 +1,136 @@
+                                   will look for dependencies. Defaults to
+                                   `~/.m2/repository`.
+      -K, --auto-cache             Create and use .lumo_cache dir for cache
+      -k, --cache path             If dir exists at path, use it for cache
+      -q, --quiet                  Quiet mode; doesn't print the banner
+      -v, --verbose                Emit verbose diagnostic output
+      -d, --dumb-terminal          Disable line editing / VT100 terminal
+                                   control
+      -s, --static-fns             Generate static dispatch function calls
+      -f, --fn-invoke-direct       Do not not generate `.call(null...)` calls
+                                   for unknown functions, but instead direct
+                                   invokes via `f(a0,a1...)`.
+      -A x, --checked-arrays x     Enables checked arrays where x is either warn
+                                   or error.
+      -n opts, --socket-repl x     Enable a socket REPL where x is port, IP:port
+                                   or JSON of the following form, where port is
+                                   required:
+                                   {"host":   "localhost",
+                                    "port":   12345,
+                                    "accept": "some.namespaced.clojure/fn",
+                                    "args":   ["args", {"for": "the accept fn"}]}
+
+    main options:
+      -m ns-name, --main=ns-name   Call the -main function from a namespace
+                                   with args
+      -r, --repl                   Run a repl
+      path                         Run a script from a file or resource
+      -                            Run a script from standard input
+      -h, -?, --help               Print this help message and exit
+      -l, --legal                  Show legal info (licenses and copyrights)
+
+    The init options may be repeated and mixed freely, but must appear before
+    any main option.
+
+    Paths may be absolute or relative in the filesystem.
+
+Using ClojureScript Libraries
+-----------------------------
+
+Lumo can make use of ClojureScript libraries, making the code in such
+libraries available to your programs.
+
+However, Lumo is not part of the Java ecosystem, the way Clojure is.
+Lumo does not run Java code, that means it can't directly access Maven
+artifacts on the internet, such as downloading ClojureScript JARs from
+`Clojars <https://clojars.org>`_. [#maven]_
+
+Using Downloaded Libraries
+~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The ``-c`` (``--classpath``) option is used to specify where to search for
+source code as either directories, or packaged inside JAR files.
+
+For example, if you have previously downloaded the
+`clojure/tools.cli <https://github.com/clojure/tools.cli>`_ package,
+you can store it in a local project folder and tell Lumo about it::
+
+  $ lumo -c src:lib/tools.cli-0.3.5.jar -m roll.dice --json 3d6
```

Comment:
I must be blind but I don't think I see the `roll.dice` source file here, should we show it as part of the doc? I would like to add it to the wiki as well.

Reply:
It's hypothetical ... I never even wrote it.